src/compiler/utilities.ts

```diff
@@ -4111,7 +4113,9 @@ namespace ts {
         return (ch >= CharacterCodes.a && ch <= CharacterCodes.z) || stringContains((name as string), "-") || stringContains((name as string), ":");
     }
 
+    /** @deprecated @internal*/

```

Comment:
No need to have `@internal`, it's not exported.

Reply:
removed tag